### DISCUSSION TITLE
Is there a plan to address the issues raised in the Radash library?

### DISCUSSION THREAD

**Discussion by AQian0** [radashi-org/radashi#45] [Ideas]
If you think it would be beneficial, I think I can work on it experimentally now without waiting for a new issue to come forward.

**Comment by aleclarson** [comment]
Absolutely, I've already addressed a lot of issues and unmerged PRs. Not everything will be addressed, since my time is limited and many of the issues/PRs are not aligned with Radashi's vision. Therefore, you should mention here what you'd like to tackle before doing it, so you don't waste much time working against the needs of the project! :)

Improvements to function performance are always welcome, as long as they are non-negligible and you keep the code readable/approachable.

You can use our PR labels to see what's been done already:
- [New features](https://github.com/radashi-org/radashi/pulls?q=is%3Apr+label%3A%22new+feature%22)  
  Any PR that adds a new function or extends the functionality of an existing one.
- [Upstream fixes](https://github.com/radashi-org/radashi/pulls?q=is%3Apr+label%3A%22upstream+fix%22)  
  Any PR that fixes the code we inherited from Radash is tagged as an “upstream fix”.
- [Performance improvements](https://github.com/radashi-org/radashi/pulls?q=is%3Apr+label%3A%22%F0%9F%94%A5+performance%22)  
  Any PR that improves the performance of a Radashi function.
- [Maintenance improvements](https://github.com/radashi-org/radashi/pulls?q=is%3Apr+label%3Amaintenance)  
  Any PR that makes it easier to maintain the quality of the codebase.

Every PR that's merged is already useable by installing `radashi@beta`. The only exception is if the PR was merged today, since the `beta` tag is only updated once per day at 5:00AM UTC.

Lastly, I've improved the Github workflows to enable automatic `beta` releases, as well as “preview releases” that let the community publish PRs to NPM without waiting for maintainers to review and merge them.
